docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+group capability.
+
+Note that in production networks, it will not be possible or desirable for one
+user to be able to update all of these channels (and parts of configurations)
+unilaterally. The orderer system channel, for example, is administered exclusively
+by ordering organization admins (though it is possible to add peer organizations
+as ordering service organizations). Similarly, updating either the `Orderer` or
+`Channel` groups of a channel configuration requires the signature of an ordering
+service organization in addition to peer organizations. Distributed systems
+require collaborative management.
+
+### Prerequisites and considerations
+
+If you haven’t already done so, ensure you have all of the dependencies on your
+machine as described in [Prerequisites](./prereqs.html). This will ensure that
+you have the latest versions of the tools required to make a channel
+configuration update.
+
+Although Fabric binaries can and should be upgraded in a rolling fashion, it is
+important to **finish upgrading binaries before enabling capabilities**. Any binaries
+which are not upgraded to at least the level of the relevant capabilities will
+crash to indicate a misconfiguration which could otherwise result in a ledger
+fork.
+
+Once a capability has been enabled, it becomes part of the permanent record for
+that channel. This means that even after disabling the capability, old binaries
+will not be able to participate in the channel because they cannot process
+beyond the block which enabled the capability to get to the block which disables
+it. As a result, once a capability has been enabled, disabling it is neither
+recommended nor supported.
+
+For this reason, think of enabling channel capabilities as a point of no return.
+Please experiment with the new capabilities in a test setting and be confident
+before proceeding to enable them in production.
+
+#### Create a capabilities config file
+
+Note that this tutorial presumes that a file called `capabilities.json` has been
+created and includes the capability updates you want to make to the various
+sections of the config. It also uses `jq` to apply the edits to the modified
+config file.
+
+Note that you are not obligated to create a file like `capabilities.json` or to
+use a tool like `jq`. The modified config can also be edited manually (after it
+has been pulled, translated, and scoped). Check out this
+[sample channel configuration](./config_update.html#sample-channel-configuration)
+for reference.
+
+However, the process described here (using a JSON file and a tool like `jq`) does
+have the advantage of being scriptable, making it suitable for proposing
+configuration updates to a large number of channels. This is why it is **the
+recommended way to update channels**.
+
+In this example, the ``capabilities.json`` file looks like this (note: if you
+are updating your channel as part of [Upgrading to the latest version of Fabric](./upgrade_to_newest_version.html)
```

Comment:
I would but the capabilities information, and links to the key concept topic, and perhaps an overview of what the capabilities mean so folks have more context when they are making and editing this file.

Reply:
The doc begins with the link to the capabilities key concept doc.